extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/CamelConfig.java

```diff
@@ -212,25 +212,15 @@
     public static class ResourcesConfig {
 
         /**
-         * A comma separated list of Ant-path style patterns to match resources
-         * that should be <strong>excluded</strong> from the native executable. By default,
-         * resources not selected by quarkus itself are ignored. Then, inclusion
-         * of additional resources could be triggered with
-         * <code>includePatterns</code>. When the inclusion patterns is too
-         * large, eviction of previously selected resources could be triggered
-         * with <code>excludePatterns</code>.
+         * Replaced by {@code quarkus.native.resources.excludes} in Camel Quarkus 2.0.0.
+         * Using this property throws an exception at build time.
          */
         @ConfigItem
         public Optional<List<String>> excludePatterns;
 
         /**
-         * A comma separated list of Ant-path style patterns to match resources
-         * that should be <strong>included</strong> in the native executable. By default,
-         * resources not selected by quarkus itself are ignored. Then, inclusion
-         * of additional resources could be triggered with
-         * <code>includePatterns</code>. When the inclusion patterns is too
-         * large, eviction of previously selected resources could be triggered
-         * with <code>excludePatterns</code>.
+         * Replaced by {@code quarkus.native.resources.includes} in Camel Quarkus 2.0.0.
+         * Using this property throws an exception at build time.
          */
         @ConfigItem
         public Optional<List<String>> includePatterns;
```

Comment:
My understanding is that we are going to break in camel-quarkus 2.0.0, so maybe we could remove those config items ?

Reply:
Yep, it is there in `docs/modules/ROOT/pages/migration-guide/2.0.0.adoc`

Reply:
So, could we remove the includePatterns field then ?

Reply:
Ah, sorry, I have not understood initially. If we remove it, users just get a warning about an unknown property. With the changes in the Processor https://github.com/apache/camel-quarkus/pull/2813/files they get a real build time failure.

Reply:
ok, got it now :+1: